cmd/process-agent/collector.go

```diff
@@ -472,6 +483,16 @@ func (l *Collector) basicRunner(c checks.Check, results *api.WeightedQueue, exit
 	}
 }
 
+func (l *Collector) shouldDropPayload(check string) bool {
+	for _, d := range l.dropCheckPayloads {
```

Comment:
Since this is on every payload, we should probably avoid the loop. It can help have this lookup done via a map, incidentally @brycekahle made a suggestion towards that.

Reply:
At that size, map lookups and linear search are comparably performant, so that sounds good.

Reply:
Good point, agreed.